main_test.go

```
@@ -0,0 +1,1482 @@
+
+// ---------------------------------------------------------------------------
+// LoadConfig tests
+// ---------------------------------------------------------------------------
+
+func writeTempConfig(t *testing.T, content string) string {
+	t.Helper()
+	f, err := os.CreateTemp("", "wol-proxy-test-*.toml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { os.Remove(f.Name()) })
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)